.github/workflows/codeql.yml

```diff
@@ -1,23 +1,14 @@
 name: "CodeQL"
 
 on:
-  push:
-    branches:
-      - master
-      - branch/*
-  pull_request:
-    branches:
-      - master
-      - branch/*
-    paths-ignore:
-      - 'docs/**'
-      - 'rfd/**'
-      - '**.md'
+  schedule:
+    - cron: '0 13 * * *' # At 1:00 PM UTC every day
 
 jobs:
   analyze:
     name: Analyze
-    runs-on: ubuntu-22.04-32core
+    if: ${{ !startsWith(github.head_ref, 'dependabot/') }}
```

Comment:
Technically this `if` is no longer necessary, as this runs on a schedule on specific branches, and not on arbitrary PR branches.

Reply:
I considered removing it, but there is still the hope that this will resume in the PR flow some day.